packages/react-components/src/components/SendBox.tsx

```diff
@@ -233,12 +233,20 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
     }
 
     // we dont want to send empty messages including spaces, newlines, tabs
-    if (!EMPTY_MESSAGE_REGEX.test(textValue)) {
-      onSendMessage && onSendMessage(textValue);
+    // Message can be empty if there is a valid file upload
+    if (!EMPTY_MESSAGE_REGEX.test(textValue) || hasFile) {
+      // Chat SDK doesn't send messages with empty content. For sending files without text message,
```

Comment:
I don't have 100% confidence in this statement.
In conversation with the SDK team to understand the exact behavior.
This is based on my tests.